src/repo.test.ts

```diff
@@ -117,4 +103,125 @@ describe('git-utils', () => {
       );
     });
   });
+
+  describe('getTagNames', () => {
+    it('returns all of the tag names that match a known format, sorted from earliest created to latest created', async () => {
```

Comment:
Nit: Update description to match change in sort order
```suggestion
    it('returns all of the tag names that match a known format, sorted by tag name in ascending order', async () => {
```

Reply:
Fixed in [8190f5f](https://github.com/MetaMask/create-release-branch/pull/23/commits/8190f5f0a7cc46ec85269ed8645c909df555e757).